website/docs/d/kubernetes_cluster.html.markdown

```diff
@@ -86,6 +86,16 @@ A `agent_pool_profile` block exports the following:
 
 * `max_pods` - The maximum number of pods that can run on each agent.
 
+* `availability_zones` - The availability zones used for the nodes. 
+
+* `enable_auto_scaling` - If the auto-scaler is enabled. 
+
+~>  **Note:** Support for the `type` of `VirtualMachineScaleSets` is currently in Public Preview on an opt-in basis. You can enable this feature using the Azure CLI by running `az feature register --name VMSSPreview --namespace Microsoft.ContainerService`
+
+* `min_count` - (Optional) Minimum number of nodes for auto-scaling 
```

Comment:
since this is a Data Source this is read-only - so could we remove this:

```suggestion
* `min_count` - Minimum number of nodes for auto-scaling 
```

Reply:
Fixed in the commit.